### PR TITLE
Enable automatic Android tests

### PR DESCRIPTION
**Pull request by ppkarwasz**
This PR is a work in progress to enable automatic tests of Log4j Core on Android.

### CODE REVIEW

File: log4j-samples-android/gradlew [hunk 1]
```diff
@@ -17,67 +17,101 @@
...
-  Darwin* )
-    darwin=true
-    ;;
-  MINGW* )
-    msys=true
-    ;;
-  NONSTOP* )
-    nonstop=true
-    ;;
+case "$( uname )" in                #(
```
Comment (vy) [inline log4j-samples-android/gradlew:110]: What are these `#(` suffixes for?

**Review by vy — CHANGES_REQUESTED** [review]
Can you implement the following changes too, please?

* ~Switch from Gradle to Maven~
* ~Make this a submodule of the `/pom.xml`~
* Add a `README.adoc`
* Add a `/README.adoc` entry (GraalVM module is missing there too)

File: log4j-samples-android/gradlew (hunk 1 above)
Comment (ppkarwasz) [reply log4j-samples-android/gradlew:110]: I don't know, maybe Gradle developers like to end each line with `#(`.

Comment (vy) [reply log4j-samples-android/gradlew:110]: Oh, I thought you added them.
I see that [the `gradlew` has it actually]( https://github.com/gradle/gradle/blob/2b5e5f3275ee87c293a9c8490b58167b7e34c208/gradlew#L111).

**Review by vy — APPROVED** [review]

**Pull request merged**